src/main/config/parameter_group_ids.h

```diff
@@ -62,6 +62,7 @@
 #define PG_CHANNEL_RANGE_CONFIG 44
 #define PG_MODE_COLOR_CONFIG 45
 #define PG_SPECIAL_COLOR_CONFIG 46
+#define PG_LED_TYPE 47
```

Comment:
@hydra: PG_ names should be normalized soon, before configurator does know about them ..